src/bundle/Resources/public/js/CKEditor/custom-tags/inline-custom-tag/inline-custom-tag-editing.js

```diff
@@ -0,0 +1,146 @@
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import { toWidget } from '@ckeditor/ckeditor5-widget/src/utils';
+import Widget from '@ckeditor/ckeditor5-widget/src/widget';
+
+import IbexaInlineCustomTagCommand from './inline-custom-tag-command';
+
+class IbexaInlineCustomTagEditing extends Plugin {
+    static get requires() {
+        return [Widget];
+    }
+
+    defineSchema() {
+        const { schema } = this.editor.model;
+
+        schema.register('inlineCustomTag', {
+            isInline: true,
+            isObject: true,
+            allowWhere: '$text',
+            allowAttributes: ['customTagName', 'values'],
+        });
+
+        schema.register('inlineCustomTagContent', {
+            isInline: true,
+            allowIn: 'inlineCustomTag',
+            allowChildren: '$text',
+        });
+    }
+
+    defineConverters() {
+        const { conversion } = this.editor;
+
+        conversion.for('editingDowncast').elementToElement({
+            model: 'inlineCustomTag',
+            view: (modelElement, { writer: downcastWriter }) => {
+                const container = downcastWriter.createContainerElement('span', {
+                    'data-ezelement': 'eztemplateinline',
+                    'data-ezname': modelElement.getAttribute('customTagName'),
+                    class: 'ez-custom-tag',
+                });
+
+                return toWidget(container, downcastWriter);
+            },
+        });
+
+        conversion.for('dataDowncast').elementToElement({
+            model: 'inlineCustomTag',
+            view: (modelElement, { writer: downcastWriter }) => {
+                const container = downcastWriter.createContainerElement('span', {
+                    'data-ezelement': 'eztemplateinline',
+                    'data-ezname': modelElement.getAttribute('customTagName'),
+                });
+                const config = downcastWriter.createUIElement('span', { 'data-ezelement': 'ezconfig' }, function(domDocument) {
+                    const domElement = this.toDomElement(domDocument);
+
+                    domElement.innerHTML = Object.entries(modelElement.getAttribute('values')).reduce((total, [attribute, value]) => {
+                        const attributeValue = value !== null ? value : '';
```

Comment:
You can use `value ?? ''` here.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator
```
The nullish coalescing operator (??) is a logical operator that returns its right-hand side operand when its left-hand side operand is null or undefined, and otherwise returns its left-hand side operand.
```